client/examples/read_to_csv_all.py

```diff
@@ -0,0 +1,75 @@
+import sys
+sys.path.append('../src/ptprobe')
+
+import datetime
+import logging
+import threading
+import time
+import argparse
+import board
+from sinks import CsvSampleSink
+
+
+ports = ["COM1", "COM2"]
```

Comment:
Once the OS ID's the qtpy's, the port seems to stay fixed. Agree that hardcoding is not pythonic/general - simply trying to abstract away more of the arguments so that whomever runs the script for it, won't have to remember what end up being static values (mostly).